server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/issue/DefaultAssigneeTest.java

```diff
@@ -58,26 +66,48 @@ public void no_default_assignee() {
   }
 
   @Test
-  public void default_assignee() {
+  public void set_default_assignee() {
     settings.setProperty(CoreProperties.DEFAULT_ISSUE_ASSIGNEE, "erik");
-    when(userIndex.getNullableByLogin("erik")).thenReturn(new UserDoc().setLogin("erik").setActive(true));
+    UserDto userDto = db.users().insertUser("erik");
+    db.organizations().addMember(organizationDto, userDto);
 
     assertThat(underTest.getLogin()).isEqualTo("erik");
   }
 
   @Test
   public void configured_login_does_not_exist() {
     settings.setProperty(CoreProperties.DEFAULT_ISSUE_ASSIGNEE, "erik");
-    when(userIndex.getNullableByLogin("erik")).thenReturn(null);
 
     assertThat(underTest.getLogin()).isNull();
   }
 
   @Test
   public void configured_login_is_disabled() {
     settings.setProperty(CoreProperties.DEFAULT_ISSUE_ASSIGNEE, "erik");
-    when(userIndex.getNullableByLogin("erik")).thenReturn(new UserDoc().setLogin("erik").setActive(false));
+    db.users().insertUser(user -> user.setLogin("erik").setActive(false));
 
     assertThat(underTest.getLogin()).isNull();
   }
+
+  @Test
+  public void configured_login_is_not_member_of_organization() {
```

Comment:
remark: in a multi org instance, the global default issue assignee shouldn't exist. It should be set on each organization.
I know properties are not organization ready